server/accounts.go

```diff
@@ -2871,8 +2871,35 @@ func (s *Server) updateAccountClaimsWithRefresh(a *Account, ac *jwt.AccountClaim
 			alteredScope[k] = struct{}{}
 		}
 	}
+	// collect mappings that need to be removed
+	removeList := []string{}
+	for _, m := range a.mappings {
+		if _, ok := ac.Mappings[jwt.Subject(m.src)]; !ok {
+			removeList = append(removeList, m.src)
+		}
+	}
 	a.mu.Unlock()
 
+	for sub, wm := range ac.Mappings {
+		mappings := make([]*MapDest, len(wm))
+		for i, m := range wm {
+			if m.Weight == 0 {
+				m.Weight = 100
+			}
+			mappings[i] = &MapDest{
+				Subject:    string(m.Subject),
+				Weight:     m.Weight,
+				OptCluster: m.OptCluster,
```

Comment:
Is OptCluster my name or yours? Could just be Cluster and be allowed to be empty..

Reply:
Both OptCluster in MapDest is your name and m.OptCluster is mine (in jwt comitted)
It is allowed to be empty.
Want me to change it to Cluster in jwt?

Reply:
Yea I think in both spots..

Reply:
fixed